MAUI/Autocomplete/UI-Customization.md

```diff
@@ -11,6 +11,10 @@ documentation: ug
 
 This section explains different UI customizations available in the [.NET MAUI Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) control.
 
```

Comment:
@ManjudhanasekaranSF4622 Can you add the task id?

Reply:
yes sir.